app/packages/operators/src/built-in-operators.ts

```diff
@@ -1155,11 +1155,13 @@ export class SetActiveFields extends Operator {
   useHooks(): {
     setActiveFields: (fields: string[]) => void;
   } {
+    // NOTE: useRecoilValue(fos.modal) is always false here
```

Comment:
@benjaminpkane this seems like a regression in the behavior of `fos.modal` - what I have here is a hack (using currentSample). Any idea what is going on?